test/lib/ufe/testDuplicateCmd.py

```diff
@@ -161,3 +162,36 @@ def childrenNames(children):
 
         self.assertIn(sphereDupItem, worldChildren)
         self.assertIn(ball35DupItem, propsChildren)
+        """
+
+        # The duplicated items should not be assigned to the name of a
+        # deactivated USD item.
+
+        cmds.select(clear=True)
+
+        # Delete the even numbered props:
+        evenPropsChildrenPre = propsChildrenPre[0:35:2]
+        for propChild in evenPropsChildrenPre:
+            ufe.GlobalSelection.get().append(propChild)
+        cmds.delete()
+
+        worldHierarchy = ufe.Hierarchy.hierarchy(worldItem)
+        worldChildren = worldHierarchy.children()
+        propsHierarchy = ufe.Hierarchy.hierarchy(propsItem)
+        propsChildren = propsHierarchy.children()
+        propsChildrenPostDel = propsHierarchy.children()
+
+        # Duplicate Ball_1, which should create Ball_36, not Ball_2
```

Comment:
Kinda feels like you should check this, right after line 191, no?  In the spirit of moving assertions out of comments and into code.

Reply:
Are you suggesting an explicit string check to verify that it's making Ball_36?  That's probably reasonable.  My primary concern was just to avoid using Ball_2 (or any of the even-numbered prop names, since they've been deactivated).

Reply:
Exactly, yes, and you can then remove the comment.

Reply:
I added the test.  For clarify, I left the portion of the comment explaining that I'm duplicating Ball_1, but eliminated the part describing what I hoped it would do -- enforcing that by the assertion.